.github/workflows/doc-tests.yaml

```
@@ -2,20 +2,33 @@ name: Lint (Docs)
 run-name: Lint (Docs)
 on:
   pull_request:
-    paths:
-      - 'CHANGELOG.md'
-      - 'docs/**'
-      - 'examples/**'
+
   merge_group:
-    paths:
-      - 'docs/**'
-      - 'examples/**'
-      - 'CHANGELOG.md'
+
 
 jobs:
+  changes:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    outputs:
+      changed: ${{ steps.changes.outputs.changed }}
+    steps:
+      - uses: dorny/paths-filter@4067d885736b84de7c414f582ac45897079b0a78
```

```suggestion
      - uses: dorny/paths-filter@4067d885736b84de7c414f582ac45897079b0a78 # v2.11.1
```

Dependabot is smart enough to keep these comments up to date, and it makes it a bit ealier to track which version we're on.  See https://github.com/gravitational/teleport/pull/31719/files#diff-89bb7fe01ada50f10d5a233e5e4336510150c4f76b6bb904394b0e5017165de7R25 for an example.

---

Nice, thanks!

---

Had to back up a few commits to land on v2.11.1, but that's fine.